src/openrct2/rct1/S4Importer.cpp

```diff
@@ -1705,11 +1705,11 @@ class S4Importer final : public IParkImporter
 
     void ImportMoneyEffect(MoneyEffect* dst, MoneyEffect* src)
     {
-        dst->move_delay = src->move_delay;
-        dst->num_movements = src->num_movements;
-        dst->value = src->value;
-        dst->offset_x = src->offset_x;
-        dst->wiggle = src->wiggle;
+        dst->MoveDelay = src->MoveDelay;
```

Comment:
This isn't right. S4 should have a RCT12SpriteMoneyEffect. It's not a mistake with this pr though.

Reply:
I don't think it is a mistake, just we haven't got round to changing it yet. They all used to use the same structs, we have gradually been copying them and removing the push, 1 pop.

Reply:
Yes but for now we don't enforce packing on money effect so who knows if this works atm.

Reply:
I created #11757 to track this issue, feel free to add more details there.